examples/plot_simulate_evoked.py

```diff
@@ -42,12 +42,17 @@
 import matplotlib.pyplot as plt
 fig, axes = plt.subplots(2, 1, sharex=True, figsize=(6, 6))
 for dpl in dpls:
-    dpl.plot(ax=axes[0], layer='agg')
+    dpl.plot(ax=axes[0], layer='agg', show=False)
 net.plot_input(ax=axes[1])
 
 ###############################################################################
-# Finally, we can also plot the spikes.
-net.plot_spikes()
+# Also, we can plot the spikes and write them to txt files.
+# Note that we can use formatting syntax to specify the filename pattern
+# with which each trial will be written. To read spikes back in, we can use
+# wildcard expressions.
+net.spikes.write('spk_%d.txt')
+spikes = read_spikes('spk_*.txt')
```

Comment:
could we write this to a temporary directory to avoid messing the current directory that the user is in? You can use something here: https://docs.python.org/3/library/tempfile.html. Maybe `mkdtemp`